experimental/dds/tree2/src/test/feature-libraries/node-key/nodeKeyIndex.bench.ts

```diff
@@ -104,10 +104,10 @@ describe("Node Key Index Benchmarks", () => {
 							const ids: (LocalNodeKey | undefined)[] = [];
 							for (let i = 0; i < nodeCount; i++) {
 								const nodeKey =
-									i % period === 0 ? tree.nodeKey.generate() : undefined;
+									i % period === 0 ? tree.view.nodeKey.generate() : undefined;
```

Comment:
Not necessarily worth fixing, but seems like it might have been cleaner to make the `tree` variable point to `tree.view` instead of adding an extra dereference to all these lines.